pkg/dbutil/index.go

```diff
@@ -120,19 +120,32 @@ func SelectUniqueOrderKey(tbInfo *model.TableInfo) ([]string, []*model.ColumnInf
 
 	for _, index := range tbInfo.Indices {
 		if index.Primary {
```

Comment:
`if index.Primary || index.Unique `

Reply:
I preferred to use primary key  as order by key when had primary key and unique key